CHANGELOG.md

```diff
@@ -24,6 +40,12 @@ to [Semantic Versioning](http://semver.org/).
   their collapsed/inactive content would be hidden.)
 - The Clubs page no longer makes superfluous network requests to
   retrieve the Clubs list.
+- Only .jpg, .png, and .svg files in the `/img` directory are being watched
+  (when running `npm start`) and copied to the `/dist` directory.
+- Ionicons icon font has been dropped for FontAwesome.
+- Hero unit background images are positioned at center top. This improves
+  focus on the main visual interests on the photographs.
```

Comment:
Maybe add ", especially on mobile viewports" at the end of the sentence? Up to you.